app/views/partials/_navbar.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="bk-nav bk-nav-mobile border-top">
```

Comment:
I would prefer if this were an actual `<nav>` HTML tag to ensure more semantic markup.